packages/cli-main/oclif.manifest.json

```diff
@@ -1 +1 @@
-{"version":"3.26.0","commands":{"editions":{"id":"editions","description":"Shopify editions","strict":true,"pluginName":"@shopify/cli","pluginAlias":"@shopify/cli","pluginType":"core","hidden":true,"aliases":[],"flags":{},"args":[]},"kitchen-sink":{"id":"kitchen-sink","description":"View all the available UI kit components","strict":true,"pluginName":"@shopify/cli","pluginAlias":"@shopify/cli","pluginType":"core","hidden":true,"aliases":[],"flags":{},"args":[]},"logs":{"id":"logs","description":"View full debug logs from the Shopify CLI","strict":true,"pluginName":"@shopify/cli","pluginAlias":"@shopify/cli","pluginType":"core","aliases":[],"flags":{"preset":{"name":"preset","type":"option","description":"The preset to apply to the current command.","hidden":true,"multiple":false},"verbose":{"name":"verbose","type":"boolean","description":"Increase the verbosity of the logs.","hidden":false,"allowNo":false},"last-command":{"name":"last-command","type":"boolean","description":"View logs for the most recent command only","hidden":true,"allowNo":false}},"args":[],"_globalFlags":{}},"upgrade":{"id":"upgrade","description":"Upgrade the Shopify CLI","strict":true,"pluginName":"@shopify/cli","pluginAlias":"@shopify/cli","pluginType":"core","aliases":[],"flags":{"path":{"name":"path","type":"option","description":"The path to your project directory.","hidden":false,"multiple":false}},"args":[],"_globalFlags":{}},"version":{"id":"version","description":"Shopify CLI version","strict":true,"pluginName":"@shopify/cli","pluginAlias":"@shopify/cli","pluginType":"core","aliases":[],"flags":{},"args":[]},"auth:logout":{"id":"auth:logout","description":"Logout from Shopify","strict":true,"pluginName":"@shopify/cli","pluginAlias":"@shopify/cli","pluginType":"core","aliases":[],"flags":{},"args":[]}}}
\ No newline at end of file
+{"version":"3.27.0","commands":{}}
```

Comment:
Is this expected @Arkham

Reply:
@pepicrft we had a bug in our nx config :)